sdk/src/test/tutorial/java/com/finbourne/lusid/tutorials/ibor/Bitemporal.java

```diff
@@ -7,6 +7,7 @@
 import com.finbourne.lusid.model.TransactionRequest;
 import com.finbourne.lusid.model.UpsertPortfolioTransactionsResponse;
 import com.finbourne.lusid.model.VersionedResourceListOfTransaction;
+import parity.LusidFeature;
```

Comment:
move this to java/com/finbourne/features so that its in the com.finbourne.features namespace

Reply:
it now imports it from com.finbourne.features instead of parity. Pushed the change